install_config/aggregate_logging.adoc

```diff
@@ -1088,7 +1071,7 @@ oc logs -f /var/log/fluentd/fluentd.log
 
 Fluentd writes logs to a specified file, by default `/var/log/fluentd/fluentd.log`, or to the console, based on the `LOGGING_FILE_PATH` environment variable.  
 
-To change the default output location for the Fluentd logs, use the `LOGGING_FILE_PATH` parameter 
+To set the output location for the Fluentd logs, use the LOGGING_FILE_PATH` parameter 
 in the xref:../install/configuring_inventory_file.html#configuring-ansible[default inventory file]. 
```

Comment:
LOGGING_FILE_PATH is a fluentd Environment variable. It couldn't be used as inventory variable

Reply:
@mburke5678  
1) The fluentd log couldn't be sent to STDOUT.  When LOGGING_FILE_PATH=console, It  send log to a file named console rather than STDOUT.
2) You can use the command  "oc set env ds/logging-fluentd LOGGING_FILE_PATH=xxx" to change the fluentd Environment.

Reply:
@anpingli Thank you for catching this. I made the changes in https://github.com/openshift/openshift-docs/pull/13259, as this PR had a number of merge issues.